tests/monitors/kubernetes_cluster/kubernetes_cluster_test.py

```diff
@@ -205,3 +205,29 @@ def test_cronjobs(k8s_cluster):
                 ),
                 timeout_seconds=300,
             )
+
+
+CONTAINER_COMPUTE_RESOURCE_METRICS = [
+    "kubernetes.container_cpu.limit",
+    "kubernetes.container_cpu.request",
+    "kubernetes.container_memory.limit",
+    "kubernetes.container_memory.request",
+]
+
+
+@pytest.mark.kubernetes
+def test_container_compute_resouorce_metrics(k8s_cluster):
```

Comment:
Expand the integration tests to actually test for container dimensions, esp for the logic that strips off the `docker://` prefix.

You can use the K8s Python client to get the running pods and see what the expected pod names and container names should be and then assert that you are getting datapoints with those exact dimensions.